### PR TITLE
Update helm2 version to 2.17.0

### PR DESCRIPTION
**Pull request by jenting**
switch helm stable chart to use the new location.

**What this PR does / why we need it**:

Update helm2 version to 2.17.0 to switch helm stable chart to use the new location.

**Which issue this PR fixes** *

N/A

**Special notes for your reviewer**:

N/A

### CODE REVIEW

**Review by cpanato — APPROVED** [review]
/lgtm

**Comment by ktham** [comment]
Any chance this version bump fix can be released soon?

**Review by davidkarlsen — APPROVED** [review]

File: pkg/util/util_test.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func TestFlatten(t *testing.T) {
 	}
 
 	for index, testData := range testDataSlice {
-		t.Run(string(index), func(t *testing.T) {
+		t.Run(fmt.Sprint(index), func(t *testing.T) {
```
Comment (davidkarlsen) [inline pkg/util/util_test.go:36]: heading says version upgrade, how are these changes related?

Comment (jenting) [reply pkg/util/util_test.go]: I'll create another PR to address this. Revert it.

Comment (cpanato) [reply pkg/util/util_test.go]: thanks

**Review by cpanato — APPROVED** [review]

**Comment by jenting** [comment]
> Any chance this version bump fix can be released soon? (our helm linting job is failing, not a big deal, but definitely would be awesome to see this rolled out!  )

@ktham FYR https://github.com/helm/chart-testing/pull/319
